ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -16,6 +16,7 @@
 #define MAX_TERM_LENGTH 245
 
 #define XAPIAN_PREFIX_EXACT_TITLE "XEXACTS"
+#define XAPIAN_PREFIX_CONTENT_TYPE "XCONTENTTYPE"
```

Comment:
I believe this will have to be changed to `T` -- see the commend in the other PR